assets/pages/new-contributions-landing/components/ContributionAmount.jsx

```diff
@@ -37,15 +47,18 @@ const formatAmount = (currency: Currency, spokenCurrency: SpokenCurrency, amount
     `${amount.value} ${amount.value === 1 ? spokenCurrency.singular : spokenCurrency.plural}` :
     `${currency.glyph}${amount.value}`);
 
-const renderAmount = (currency: Currency, spokenCurrency: SpokenCurrency) => (amount: Amount, i) => (
+const renderAmount = (currency: Currency, spokenCurrency: SpokenCurrency, props: PropTypes) => (amount: Amount) => (
   <li className="form__radio-group-item">
     <input
       id={`contributionAmount-${amount.value}`}
       className="form__radio-group-input"
       type="radio"
       name="contributionAmount"
       value={amount.value}
-      checked={i === 0}
+      /* eslint-disable react/prop-types */
+      checked={props.amount && amount.value === props.amount.value}
```

Comment:
Another question for @Ap0c: should we have `react/prop-types` as a lint rule? From what I can tell from [the docs](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prop-types.md), it looks like it's designed to work with React's [own little typechecking mechanism](https://reactjs.org/docs/typechecking-with-proptypes.html), which from what I can tell is irrelevant if you have Flow or Typescript

Reply:
I think the linter has become confused here because it's seeing the word `props` used in a strange context. This usage isn't actually referring to the props of this component here. If you rename the parameter to something else the error disappears.